lib/ressources/theme/green/theme_green_light.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 
-Color lightGreen = const Color(0xff83f28f);
+Color lightGreen = const Color(0xff65793C);
```

Comment:
I think we can create a Color class, is that we will be able to access the colors from this class anywhere and change them when we need, Like : 
`import 'package:flutter/material.dart';

class ColorSelect {
  static final lightGreen = const Color(0xff65793C);
}`